CryptoPy/cryptoScraper.py

```diff
@@ -1,34 +1,54 @@
-# API used = Nomics
+# API used = CryptoCompare
 
 import requests
 from requests.exceptions import ConnectionError, Timeout, TooManyRedirects
+import csv
+import asyncio
 
 
-url = 'https://api.nomics.com/v1/currencies/ticker'
+url_for_minute_data = 'https://min-api.cryptocompare.com/data/histominute'
+url_for_hour_data = 'https://min-api.cryptocompare.com/data/histohour'
 
-key = 'ENTER YOUR API KEY'
+# curreny symbols: 'BTC' for bitcoin, 'ETH' for ethereum, 'LTC' for litecoin and 'XRP' for ripple
 Currency = 'BTC'
 
 
-# enter your api key from nomics in "key"
-# curreny symbols: 'BTC' for bitcoin, 'ETH' for ethereum, 'LTC' for litecoin and 'XRP' for ripple
-def getPrice(key, currencySymbol, interval='1hr'):
-    parameters = {
-        'key': key,
-        'ids': currencySymbol,
-        'interval': interval,
-        'convert': 'USDT'
-    }
-
-    try:
-        response = requests.get(url, params=parameters)
-        data = response.json()
-        price = [dp["price"] for dp in data]
-        return(', '.join(price))
-
-    except (ConnectionError, Timeout, TooManyRedirects) as e:
-        print(e)
-
-
-newPrice = getPrice(key, Currency)
-print(newPrice)
+def write_to_file(symbol, data):
+    with open('Data/BTCUSDT_sample_data.csv', mode='a', newline='') as data_file:
+        data_writer = csv.writer(data_file, delimiter=',', quotechar='"', quoting=csv.QUOTE_MINIMAL)
+        data_writer.writerow([data["time"], data["open"], data["high"], data["low"], data["close"], data["volumefrom"], data["volumeto"]])
```

Comment:
Rename `date` to `timestamp` and add another column next to `timestamp` and name it `date` which uses datetime. Use a simple epoch time converter in case the API doesn't provide datetime